osgi/jta/src/main/resources/jbossts-properties.xml

```diff
@@ -90,7 +98,6 @@
     -->
     <entry key="RecoveryEnvironmentBean.recoveryModuleClassNames">
         com.arjuna.ats.internal.arjuna.recovery.AtomicActionRecoveryModule
-        com.arjuna.ats.internal.txoj.recovery.TORecoveryModule
```

Comment:
Given it's not exposed to the OSGi world, I don't see the need to add this recovery module.
If we want to support TX4j in OSGi, we need to add everything needed, including a way to access this API from other OSGi bundles.